netty/src/main/java/io/grpc/netty/NettyServerBuilder.java

```diff
@@ -81,6 +81,8 @@
   private int maxHeaderListSize = GrpcUtil.DEFAULT_MAX_HEADER_LIST_SIZE;
   private long keepAliveTimeInNanos =  DEFAULT_SERVER_KEEPALIVE_TIME_NANOS;
   private long keepAliveTimeoutInNanos = DEFAULT_SERVER_KEEPALIVE_TIMEOUT_NANOS;
+  private boolean permitKeepAliveWithoutCalls;
+  private long permitKeepAliveTimeInNanos = TimeUnit.MINUTES.toNanos(5);
```

Comment:
Meh. I think it makes the code less readable by adding a layer of indirection, and it is only used in this one place. It's not like on client-side where we have multiple implementations and we want to keep them in sync. I'd rather the constant be added when it is helpful/necessary instead of now.